bin/pycbc_live

```diff
@@ -1100,7 +1108,7 @@ with ctx:
                                  args.max_psd_abort_distance)
                     status = False
 
-            if status is True:
+            if ifo not in skymap_only_ifos and status is True:
```

Comment:
This logic is not really what we want, because it will print "Insufficient data for %s analysis" even if a detector is a skymap-only detector.